nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutEmail.java

```diff
@@ -102,13 +91,16 @@ public void testExceptionWhenSending() {
 
         runner.assertQueueEmpty();
         runner.assertAllFlowFilesTransferred(PutEmail.REL_FAILURE);
-        assertEquals("Expected an attempt to send a single message", 1, processor.getMessages().size());
+        List<MockFlowFile> results = runner.getFlowFilesForRelationship(PutEmail.REL_FAILURE);
+        assertEquals("Expected an attempt to send a single message", 1, results.size());
     }
 
     @Test
     public void testOutgoingMessage() throws Exception {
         // verifies that are set on the outgoing Message correctly
-        runner.setProperty(PutEmail.SMTP_HOSTNAME, "smtp-host");
+        runner.setProperty(PutEmail.SMTP_AUTH, "false");
+        runner.setProperty(PutEmail.SMTP_HOSTNAME, "localhost");
+        runner.setProperty(PutEmail.SMTP_PORT, String.valueOf(port));
         runner.setProperty(PutEmail.HEADER_XMAILER, "TestingNiFi");
         runner.setProperty(PutEmail.FROM, "test@apache.org");
```

Comment:
Not sure if that's ok to use legitimate email address as test data especially as TO/FROM email address. NOt saying it's wrong, simply raising the question.